marketplace-and-integrations/README.md

```diff
@@ -8,9 +8,22 @@ description: >-
 
 Within this Umbraco DXP (Digital Experience Platform) area of the Umbraco documentation, you'll find topics related to extending Umbraco CMS via packages and integrations.
 
-Umbraco is used in solutions in both a "platform" and "composable" DXP context. With the former, Umbraco has responsibility for both content management and delivery, with extensions added as NuGet packages. In a composable DXP, Umbraco is often part of an overall architecture, focussed on the management of content exposed via APIs for delivery. In both situations, backoffice integrations are an important part of providing an intuitive, unified editor experience.
+Both packages and integrations are defined as Umbraco Packages that can be either [NuGet packages (Umbraco 10+) or packages zip files (Umbraco 8 or earlier)](https://docs.umbraco.com/umbraco-cms/extending/packages/types-of-packages)
```

Comment:
The content here is good, but I would say this distinction between "package" and "integration" is not to so important as the explanation of what we mean by a DXP in the context of Umbraco.  So I would suggest it goes below that, perhaps under a "Packages and integrations" sub-heading.

Obviously naming isn't great here, and it's a bit confusing that we talk about "packages" as anything you install into Umbraco via NuGet and "packages" as "those packages that aren't integrations".  Perhaps it could even just be the "integrations" that are called out for explicit definition, which, along with the diagram should illustrate what's important here.

Reply:
I agree! I removed the definition of packages and integrations but I don't know where to place it, because it's difficult to explain the diagram within the current structure of the page, which I like. Maybe we should consider simplifying the diagram before using it, or not using it at all. Let's talk about this Tuesday, and leave it out of the PR for now.